data/bash-completion/fwupdtool.in

```diff
@@ -100,7 +100,7 @@ _fwupdtool()
 			_show_modifiers
 		fi
 		;;
-	attach|detach|activate|verify-update)
+	attach|detach|activate|verify-update|reinstall)
```

Comment:
I think you need to modify `_fwupdtool_cmd_list` too

Reply:
Riht, sorry.